### DISCUSSION TITLE
How user can download an attachment from record detail view

### DISCUSSION THREAD

**Discussion by sergey-dme** [Budibase/budibase#2084] [Help]
Hey all,

I can't figure out how to download attachment from **Record Detail View** (not from all table records list view).
The file name in the attachment component is not active link to it.

<img width="557" alt="Screenshot 2021-07-25 at 05 19 03" src="https://user-images.githubusercontent.com/53024832/126885563-6098796e-6dfc-4aa1-b61c-f45f7c641a08.png">

Please advise.
Thanks.

**Comment by joebudi** [comment]
Hey! You can add a button, use the navigate action, and bind the URL to the attachment.

**Reply by sergey-dme** [reply]
Hey @joebudi Thank you for the idea. It works. But there are some downsides:

- It works always for only one/first file attached
- The button is always shown on the page even if there are no file attached

**Reply by joebudi** [reply]
Ahhh you are right. I will raise this with the team tomorrow

**Reply by joebudi** [reply]
A link is probably a better option but it's not great. We are working on conditional UI which will provide a better experience.

**Reply by sergey-dme** [reply]
Cool. Thanks. I think the filename as a link to it (at the upper left corner of the attachment component) would be the easiest solution in this case.
